lib/transforms/onnxextension_legalizer.cc

```diff
@@ -392,6 +392,56 @@ static std::vector<Def> ConvertMaximum(const ONNXExtensionInst* ext,
   return {*op0};
 }
 
+static std::vector<Def> ConvertMean(const ONNXExtensionInst* ext,
+                                    IRBuilder* builder) {
+  // Conver to a chain of Mean.
+  auto n = ext->GetNumOfOperands();
+  ConstantBuilder c_builder(ext->GetParent()->GetParent());
+  HLCHECK(n >= 1);
+  if (n == 1) {
+    return {ext->GetOperand(0)};
+  }
+  auto op0 = builder->CreateAdd(ext->GetName(), ext->GetOperand(0),
+                                ext->GetOperand(1));
+  for (unsigned i = 2; i < n; ++i) {
```

Comment:
for n == 2, can we use original "mean"?
Regular mean (2 operands) are commonly used, we don't want to convert to "add & div". that's not efficient. For more than 2 operands, we can work around it.